From b6d7eb52dcdb4cc24fe8d2f9a41b2b3cbf425854 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Wed, 24 Nov 2021 06:12:39 +0200
Subject: [PATCH] [SW-0} gaudi2: move dbi get offset function to common

Currently, dev_pci_dbi_cfg_offset_get function is implemented as
part of gaudi2 ARM file. Since this function is required for
both flows, move it to gaudi2 common file.

Change-Id: I593fa9eaf7ac6be3aca4c1b1bb3674791dff3ab3
---

diff --git a/arch/gaudi2/arm/common.c b/arch/gaudi2/arm/common.c
index 212dda5..6438ff0 100644
--- a/arch/gaudi2/arm/common.c
+++ b/arch/gaudi2/arm/common.c
@@ -339,11 +339,6 @@
 	return false;
 }
 
-u32 dev_pci_dbi_cfg_offset_get(void)
-{
-	return (u32)mmPCIE_DBI_BASE & 0xFFFFFFF;
-}
-
 void preboot_secondary_cpu_stop(void)
 {
 	u32 rst_ctrl = regmap_readl(mmCPU_CA53_CFG_ARM_RST_CONTROL);
diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 71872fc..465d1ed 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -1039,3 +1039,8 @@
 		regmap_writel(reg & ~bitmask, mmPCIE_WRAP_PCIE_IC_SEI_INTR_IND);
 	}
 }
+
+u32 dev_pci_dbi_cfg_offset_get(void)
+{
+	return (u32)mmPCIE_DBI_BASE & 0xFFFFFFF;
+}
